learning-progression.md

```diff
@@ -132,54 +132,97 @@ basic.showIcon(IconNames.Butterfly)
 
 ### Step 5: Conditionals
 
+#### 1. Study
+
+In the English language, a statement that begins with "if" is followed by a condition. If the condition is met, the next statement follows as a logical consequence. For example, in the sentence "If it starts to rain, then I will get wet." The condition is that "it starts to rain." When the condition becomes true, the second part of the sentence - "I will get wet" - also becomes true.
+
+In javascript, we use statements that begin with `if` in the same way. Take a look at the following code:
+
 ```javascript
-if (true) {
+if (true) { //This condition is always true, so the code in this block will run.
     basic.showIcon(IconNames.Heart)
     basic.pause(2000)
     basic.showIcon(IconNames.Butterfly)
-} else {
+} else { //Everything in this block is ignored because the first "if" statement was true.
     basic.showIcon(IconNames.Angry)
     basic.pause(2000)
     basic.showIcon(IconNames.Snake)
 }
-   
-if (false) {
+```
+
+In the above example, the statement begins with `if`, and the condition written within the parentheses is `true`. Therefore, the statements inside the first block of code are run. The `else` block can be thought of as an alternative to the `if` statement - it will only run if the condition for `if` is not met. Therefore, in this version of our code it will be ignored.
+
+_In javascript, words like `true` and `false` are special keywords reserved by the language. These keywords can only be used to represent a specific boolean value. More on booleans in the next step._
```

Comment:
Ambiguous: do all keywords represent specific boolean values, or just these two. In linguistics, this is called [anaphora](https://en.wikipedia.org/wiki/Anaphora_(linguistics)). This is one of the hardest ambiguities in human language to solve, though humans are remarkably good at it, using a rich background of context and common ground. In technical language, everything should be clear, even at the expense of repetition.